presto-common/src/test/java/com/facebook/presto/common/type/TimeZoneKeyTest.java

```diff
@@ -1,140 +1,123 @@
+
+     zoneId = "etc/gmt-08:20";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "-08:20");
+   }
+
+   @Test
+   @Named("UTC TEST")
+   public void utcZoneIdTest() {
+     String zoneId = "etc/utc+1";
+     String normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "-01:00");
+
+     zoneId = "etc/utc-1";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+01:00");
+   }
+
+   @Test
+   @Named("GREENWICH TEST")
+   public void greenwichZoneIdTest() {
+     String zoneId = "etc/greenwich+01:00";
+     String normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+01:00");
+
+     zoneId = "etc/greenwich+6";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+06:00");
+
+     zoneId = "etc/greenwich-6";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "-06:00");
+
+     zoneId = "etc/greenwich+11:23";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+11:23");
+   }
+
+   @Test
+   @Named("UNIVERSAL TEST")
+   public void universalZoneIdTest() {
+     String zoneId = "etc/universal+08:00";
+     String normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+08:00");
+
+     zoneId = "etc/universal-08:00";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "-08:00");
+
+     zoneId = "etc/universal+11:23";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+11:23");
+   }
+
+   @Test
+   @Named("UCT TEST")
+   public void uctZoneIdTest() {
+     String zoneId = "etc/uct+06:00";
+     String normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+06:00");
+
+     zoneId = "etc/uct-10:10";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "-10:10");
+
+     zoneId = "etc/uct+1";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+01:00");
+   }
+
+   @Test
+   @Named("ZULU TEST")
+   public void zuluZoneIdTest() {
+     String zoneId = "etc/zulu+06:00";
+     String normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+06:00");
+
+     zoneId = "etc/zulu-10:10";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "-10:10");
+
+     zoneId = "etc/zulu+1";
+     normalizedZoneId = TimeZoneKey.normalizeZoneId(zoneId);
+     assertEquals(normalizedZoneId, "+01:00");
+   }
+
+   @Test
+   @Named("EXCEPTION TEST")
+   public void exceptionZoneIdTest() {
+     assertThrows(TimeZoneNotSupportedException.class, () -> TimeZoneKey.normalizeZoneId("GMT-13:00"));
```

Comment:
Please keep the previous version if possible, I find it easier to read.

Reply:
I need to return the second version for test cases and keep normalizeZoneId() method private ?